data/templates/fry-money/config.yml

```diff
@@ -0,0 +1,12 @@
+name: Shut up and take my money
+link: http://knowyourmeme.com/memes/fry-take-my-money
+default:
+- SHUT UP AND
+- TAKE MY MONEY!
+aliases:
+- frymoney
+- shut-up-and-take-my-money
+- take-my-money
+regexes:
```

Comment:
I think the follow should all return matches:
-  `/magic/shut up and take my money`
-  `/magic/shut up and do that thing`
- `/magic/stop talking and take my money`